docs/scraping-receivers.md

```diff
@@ -0,0 +1,143 @@
+file is being used to generate an API for metrics recording, user settings to customize the emitted metrics and user
+documentation. The file schema is defined in 
+https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/cmd/mdatagen/metric-metadata.yaml. 
+Defining a metric in `metadata.yaml` DOES NOT guarantee that the metric will always be produced by the receiver. In
+some cases it may be impossible to fetch particular metrics from a system in a particular state. If it's known that 
+the metric is reported only under certain conditions (e.g. a specific version) of the external source, the 
+condition MUST be mentioned in the metric description.
+
+There are two categories of the metrics emitted by scraping receivers:
+
+- **Default metrics**: emitted by default, but can be disabled in user settings.
+
+- **Optional metrics**: not emitted by default, but can be enabled in user settings.
+
+### How to identify if new metric should be default or optional?
+
+There is no strict rule to differentiate default metrics from optional. As a rule of thumb, default metrics SHOULD be 
+treated as metrics essential to determine healthiness of a particular monitored entity. Optional metrics on the 
+other hand SHOULD be treated as a source of additional information about the monitored entity. 
+
+If two metrics are built based on the same data but have different representation, at least one of them MUST be 
+optional. For example:
+
+- `system.cpu.time` (CPU time reported as cumulative sum in seconds) - Default metric.  
+- `system.cpu.utilization` (fraction of CPU time spent in different states reported as gauge) - Optional metric.
+
+In this example, system CPU usage time can be emitted as two different metrics: a gauge as a fraction of 1 or 
+a cumulative sum in seconds. Potentially, each of them can be deducted from one another, that's why only one of 
+them is enabled by default.
+
+## Stability levels of scraping receivers
+
+All the requirements defined for components in [Collector README](../README.md#stability-levels) are applicable to 
+the scraping receivers as well. In addition, the following rules applied specifically to scraping metrics receivers:
+
+### In development
+
+The receiver is not ready for use. All the metrics emitted by the receiver are not finalized and can change in any way.
+
+### Alpha
+
+The receiver is ready for limited non-critical workloads. The list of emitted default metrics SHOULD be 
+considered as complete, but any changes to the `metadata.yaml` still MAY be applied.
+
+### Beta
+
+The receiver is ready for non-critical production workloads. The list of emitted default metrics MUST be
+considered as complete. Breaking changes to the emitted metrics SHOULD be applied following [the deprecation 
+process](#changing-the-emitted-metrics). All the metric and resource attribute names SHOULD be released in an 
+[OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification).
```

Comment:
Whether or not these metric names should be tracked is not settled in the specification itself.

https://github.com/open-telemetry/opentelemetry-specification/issues/2610

Reply:
Ok I'll remove this requirement for now